### ISSUE TITLE
AssertionError on custom color scheme

### ISSUE THREAD

**Issue by mixmastamyk** [prompt-toolkit/ptpython#315]
```
Traceback (most recent call last):
  File "~/.local/lib/python3.7/site-packages/ptpython/repl.py", line 276, in run_config
    namespace['configure'](repl)
  File "~/.ptpython/config.py", line 123, in configure
    repl.use_ui_colorscheme('my-colorscheme')
  File "~/.local/lib/python3.7/site-packages/ptpython/python_input.py", line 364, in use_ui_colorscheme
    self._current_style = self._generate_style()
  File "~/.local/lib/python3.7/site-packages/ptpython/python_input.py", line 384, in _generate_style
    self.ui_styles[self._current_ui_style_name])
  File "~/.local/lib/python3.7/site-packages/ptpython/style.py", line 41, in generate_style
    ui_style
  File "~/.local/lib/python3.7/site-packages/prompt_toolkit/styles/style.py", line 330, in merge_styles
    return _MergedStyle(styles)
  File "~/.local/lib/python3.7/site-packages/prompt_toolkit/styles/style.py", line 349, in __init__
    assert all(isinstance(style, BaseStyle) for style in styles)
AssertionError

Press ENTER to continue...
```

The custom color scheme is a dictionary.  I tried the classmethod `Style.from_dictionary()` but it didn't work either.

**Comment by mixmastamyk** [comment]
FYI, I went back to version 0.41 as it does everything I need and supports custom color schemes.  This is a must-have as the default scheme is unusable.  Lots of dark blue, which is close to non-visible on dark terminal backgrounds.  Strings have a white background, which just looks horrible.

I tried the vanschelven code mentioned above and while it does fix the crashes, the scheme format does not support several keys.  I think the issue has something to do with the alias names not (or only) being supported.  Couldn't find any docs and got tired of searching through source I don't understand well.

Honestly this is bad enough for long enough to make me seriously consider going back to bpython.  I'm using it already at times, it has a few less features but is much easier to change the color scheme.
